Lib/test/test_builtin.py

```diff
@@ -1756,6 +1757,15 @@ def __next__(self):
         l8 = self.iter_error(zip(Iter(3), "AB", strict=True), ValueError)
         self.assertEqual(l8, [(2, "A"), (1, "B")])
 
+    def test_zip_result_gc(self):
+        try:
+            gc.disable()
+            z = zip([[]])
+            gc.collect()
```

Comment:
Please explain that this GC collection untracks the internal "result" tuple of the zip object.